ArchiSteamFarm/Steam/Integration/ArchiWebHandler.cs

```diff
@@ -420,9 +420,9 @@ public sealed class ArchiWebHandler : IDisposable {
 				return null;
 			}
 
-			int points = pointsInfo.AsInteger(int.MinValue);
+			uint points = pointsInfo.AsUnsignedInteger(uint.MaxValue);
```

Comment:
Theoretically `uint.MaxValue` is a valid value, and should be treated as valid as well - consider using approach from [this comment](https://github.com/JustArchiNET/ArchiSteamFarm/pull/2330#discussion_r642081694) (unless @JustArchi has a better idea to implement it).

Reply:
Or just remove that check entirely - it's safe to assume 0 as a default, [like here](https://github.com/JustArchiNET/ArchiSteamFarm/blob/93af13ec47cb6cf096774b6345c1499d872ff6a6/ArchiSteamFarm/Steam/Integration/ArchiWebHandler.cs#L1541)